### PR TITLE
Site Corrections

### PR DESCRIPTION
**Pull request by lizzcondrey**
# Description
Correcting example model in readme for front github page
Trying to correct site build workflow

# Linked Items:
N/A

### Changed
- Example model in readme
- Sphinx build workflow

# Checklist:
- [x] I updated project documentation to reflect my changes.
- [x] My changes generate no new warnings.
- [x] I updated new and existing unit tests to account for my changes.
- [x] I linked the associated item(s) to be closed.
- [x] I bumped the version.
- [x] I added the labels corresponding to my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,11 +141,11 @@ model:
       acceptance:
         - scenario: onReceive
           given:
-          - The EchoService is running.
+           - The EchoService is running.
```
Comment (bunchw) [inline README.md:144]: Should the spaces here be the same?  Non-blocking, not a big deal.

**Review by bunchw — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lizzcondrey) [reply README.md:144]: Good catch, I'll double check. Copy/Paste may have dorkered.

Comment (lizzcondrey) [reply README.md:144]: No, this is the corrected spacing.

Comment (lizzcondrey) [reply README.md:144]: The example model given previously was improperly formatted in a few places, that was what this was trying to address.

**Pull request merged**